docs/design/coreclr/botr/clr-abi.md

```diff
@@ -4,17 +4,23 @@ This document describes the .NET Common Language Runtime (CLR) software conventi
 
 It describes requirements that the Just-In-Time (JIT) compiler imposes on the VM and vice-versa.
 
-A note on the JIT codebases: JIT32 refers to the original JIT codebase that originally generated x86 code and was later ported to generate ARM code. Later, it was ported and re-architected to generate AMD64 code (making its name something of a confusing misnomer). This work is referred to as RyuJIT. RyuJIT is being ported to generate ARM64 code. JIT64 refers to the legacy codebase that supports AMD64.
+A note on the JIT codebases: JIT32 refers to the original JIT codebase that originally generated x86 code and was later ported to generate ARM code. JIT64 refers to the legacy .NET Framework codebase that supports AMD64. The RyuJIT compiler evolved from JIT32, and now supports all platforms and architectures. See [this post](https://devblogs.microsoft.com/dotnet/the-ryujit-transition-is-complete) for more RyuJIT history.
 
-CoreRT refers to https://github.com/dotnet/corert runtime that is optimized for AOT. The CoreRT ABI differs in a few details for simplicity and consistency across platforms.
+[CoreRT](https://github.com/dotnet/corert) refers to an experimental runtime that is optimized for ahead-of-time compilation (AOT). The CoreRT ABI differs in a few details for simplicity and consistency across platforms. CoreRT has been superseded by [NativeAOT](https://github.com/dotnet/runtimelab/tree/feature/NativeAOT).
 
 # Getting started
 
-Read everything in the documented Windows ABI.
+Read everything in the documented Windows and non-Windows ABI documentation.
 
 AMD64: See [x64 Software Conventions](https://docs.microsoft.com/en-us/cpp/build/x64-software-conventions).
 
-ARM: See [Overview of ARM ABI Conventions"](https://docs.microsoft.com/en-us/cpp/build/overview-of-arm-abi-conventions).
+ARM: See [Overview of ARM32 ABI Conventions](https://docs.microsoft.com/en-us/cpp/build/overview-of-arm-abi-conventions).
+
+ARM64: See [Overview of ARM64 ABI conventions](https://docs.microsoft.com/en-us/cpp/build/arm64-windows-abi-conventions).
+
+ARM corporation ABI documentation is [here](https://developer.arm.com/architectures/system-architectures/software-standards/abi) and [here](https://github.com/ARM-software/abi-aa).
+
+The Linux System V x86_64 ABI is documented [here](https://software.intel.com/content/dam/develop/external/us/en/documents/mpx-linux64-abi.pdf), also [here](https://www.uclibc.org/docs/psABI-x86_64.pdf). It appears the latest version is 1.0 [here](https://github.com/hjl-tools/x86-psABI/wiki/x86-64-psABI-1.0.pdf), with ABI source material [here](https://gitlab.com/x86-psABIs/x86-64-ABI).
```

Comment:
```suggestion
The Linux and macOS x64 calling convention is documented in [System V Application Binary Interface AMD64 Architecture Processor Supplement](https://github.com/hjl-tools/x86-psABI/wiki/x86-64-psABI-1.0.pdf), with ABI source material [here](https://gitlab.com/x86-psABIs/x86-64-ABI).
```
I do not think we need multiple links to the same doc here. I would include the name of the doc in the link so that people can still find it in case the link becomes non-functional.